src/Mapbender/CoreBundle/Resources/public/mapbender.element.layertree.tree.js

```diff
@@ -276,10 +213,14 @@
                     cursor: "move",
                     update: function(event, ui) {
                         var $elm = $(ui.item);
-                        if ($elm.hasClass('themeContainer')) {
-                            self._sortTheme($elm);
+                        var type = $elm.attr('data-type');
+                        if (type === 'theme' || type === 'root') {
+                            self._updateSourceOrder();
+                            // self._sortTheme($elm);
+                        } else if (type === 'simple' || type === 'group') {
+                            self._updateSource($elm.closest('.serviceContainer'));
```

Comment:
Please use bind instead of creating a reference to the outer scope this

Reply:
Can't forward `this` via `bind` here, because the `this` in the inner loop scope is the matched `ul.layers` element. If I clobber that with the widget reference, the code would no longer function.

I'm removing the "that" though.